test/duct/core_test.clj

```diff
@@ -185,3 +191,17 @@
     (is (= (core/fold-modules (ig/init p))
            {::a 2, ::b (ig/ref ::a), ::c (ig/refset ::b)
             ::core/environment :production}))))
+
+(deftest test-module-key
+  (core/load-hierarchy)
+  (let [m {:duct.profile/base {}
+           :duct.module/routes {:route/resource :person}}
+        p (duct.core/prep-config m)]
+    (is (= :person (-> p :duct/router :routes)))))
+
+(deftest test-non-module-key-exception
+  (core/load-hierarchy)
```

Comment:
Is this line necessary?

Reply:
Yes, because `:duct.profile/base` derives from `:duct/profile` through the duct_hierarchy